### DISCUSSION TITLE
Evaluation format/deadline

### DISCUSSION THREAD

**Discussion by sunsonjay** [DICOMWG26AnnotationAHG/hackathon#9] [Q&A]
Hi,
Our team is curious to learn about the hackathon evaluation format of the project and the deadline? Thanks!

**Comment by sunsonjay** [comment]
Hi @hackermd Dr. Hermann, how would our project be evaluated? At the Jun 18th panel discussion? Will we get a virtual event link to it? Many thanks!

**Reply by hackermd** [reply]
@sunsonjay thank you for your note and apologies for my delayed reply. We experienced difficulties in evaluating projects given the information available in the GitHub issues and we ultimately decided to not host individual project presentations at the conference. However, we will introduce the teams during the panel and give a brief overview of the different projects that were proposed. Furthermore, we are going to continue the Hackathon event after the conference and re-assess the project proposal and evaluation process based on the experience gained so far.

**Reply by sunsonjay** [reply]
Thank you very much for your reply, for the brief overview and team intro, let me know in what format would you prefer us to provide, i.e. slide deck, video demo etc.

**Reply by sunsonjay** [reply]
Or a poster?
